oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TopNRecordsQueryDAO.java

```diff
@@ -43,8 +45,11 @@ public H2TopNRecordsQueryDAO(JDBCHikariCPClient h2Client) {
 
     @Override
     public List<SelectedRecord> readSampledRecords(final TopNCondition condition,
-                                                   final String valueColumnName,
+                                                   String valueColumnName,
                                                    final Duration duration) throws IOException {
+        if (H2_RESERVED_WORDS.contains(valueColumnName)) {
+            valueColumnName = String.format("`%s`", valueColumnName);
+        }
```

Comment:
These are duplicated.codes, could you move them to a method named formatColumnName?